.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
please refrain from adding with `git add -A` and remove this file

Reply:
Same with the .DS_Store

Reply:
👍